lib/rucio/common/utils.py

```diff
@@ -27,7 +27,7 @@
 import re
 import signal
 import socket
-import subprocess
+import subprocess  # noqa: S404
```

Comment:
We're ignoring this rule everywhere in our codebase - [S404](https://docs.astral.sh/ruff/rules/suspicious-subprocess-import/), Do you think it would make sense to just add it into the ignore list for ruff? 
Also according to the [documentation](https://docs.astral.sh/ruff/rules/suspicious-subprocess-import/), it is unstable and in preview (which we disabled), so I am not sure if this rule is enforced after disabling preview. Can you please re-run ruff and check if this is the case?